src/sqlfluff/core/parser/segments/raw.py

```diff
@@ -11,8 +11,9 @@ class RawSegment(BaseSegment):
     """This is a segment without any subsegments."""
 
     type = "raw"
-    _is_code = False
+    _is_code = True
```

Comment:
Why this change?

Reply:
Just ease. It's more likely to be true than false for any given segment.

tbh, this whole section might get trashed when we do the `ProtoSegment`.